kitsune/sumo/static/sumo/js/wiki.dashboard.js

```diff
@@ -30,8 +30,8 @@
 
     if ($('body').is('.localization-dashboard')) {
       // Add's datepicker to the create announcement pop-up
-      addDatePicker('#id_show_after');
-      addDatePicker('#id_show_until');
+      $('#id_show_after').datepicker('option', 'dateFormat', 'yy-mm-dd');
```

Comment:
i'm not sure why you deleted the `addDatePicker` function. you should have just modified that.